src/buttons/buttons.js

```diff
@@ -1,12 +1,12 @@
 angular.module('mm.foundation.buttons', [])
 
 .constant('buttonConfig', {
-    activeClass: 'active',
+    deactiveClass: 'hollow',
```

Comment:
if you want to keep the name as hollowClass it ok, by me; But I changed it to deactive because represent the state of the object which is deactivated (sorry I meant deactivated)